community.md

```diff
@@ -27,11 +27,13 @@ menu: "community"
 
 *Active committers on Petals ESB and related projects.*
 
-* Alexandre Lagane (Linagora) - [@alagane](https://github.com/alagane)
-* Bertrand Escudié (Linagora) - [@bescudie](https://github.com/bescudie)
-* Christophe Chevalier (Linagora) - [@christophechevalier](https://github.com/christophechevalier)
-* [Christophe Deneux](https://www.linkedin.com/in/cdeneux/) (Linagora) - [@chrisdeneux](https://twitter.com/ChrisDENEUX) - [@cdeneux](https://github.com/cdeneux)
-* [Pierre Souquet](https://fr.linkedin.com/in/pierre-souquet-18955528) (Linagora) - [@psouquet](https://github.com/psouquet)
+* [Alexandre Lagane](https://www.linkedin.com/in/alexandrelagane) (Linagora) - [![GitHub](./resources/images/github_mark_32px.png "GitHub")](https://github.com/alagane){:target="_blank"}
```

Comment:
Perhaps can you add the role of everyone inside the Petals team ?

Reply:
Why do you add `target: _blank`?
And roles can evolve. This is useless.

Reply:
This will open it in a new tab. I find it more practical when it comes to opening an external link.  
I didn't push roles ?!